examples/utilities.f90

```diff
@@ -17,6 +17,7 @@ module decomp_2d_testing
       module procedure decomp_2d_testing_init_2int
       module procedure decomp_2d_testing_init_5int
       module procedure decomp_2d_testing_init_6int
+      module procedure decomp_2d_testing_init_8int
```

Comment:
I feel like this is a design issue. Perhaps we could have a single init subroutine that accepts an array of arguments and the calling code maps these into 2decomp values?

Reply:
In my opinion, a proper design could be to use command-line arguments such as 
```
mpirun -n 5 test --nx 32 --ny 64 --nz 128 --prow 1 --pcol 5 ....
```
However I am not sure it can be done easily without introducing extra dependencies...

I would suggest to look at this in a dedicated issue / PR

Reply:
yeah, not suggesting to fix here, it just prompted the thought